core/libraries/messages/EE_Messages_Generator.lib.php

```diff
@@ -252,7 +254,7 @@ protected function _reset_current_properties()
      * @throws EE_Error
      * @throws InvalidArgumentException
      * @throws \EventEspresso\core\exceptions\InvalidDataTypeException
```

Comment:
why not import this class too?

Reply:
Ya I think this was automatically added by my IDE and I didn't notice it.  I decided to do a review of the entire file for lint (warning) suggestions.